addons/medical/scripts/Game/ACE_Medical/Components/Damage/SCR_CharacterDamageManagerComponent.c

```diff
@@ -4,8 +4,8 @@
 //! Add methods for interacting with pain hit zone.
 modded class SCR_CharacterDamageManagerComponent : SCR_DamageManagerComponent
 {
-	[Attribute(defvalue: "600", desc: "Time to regenerate this hit zone fully in second chance state [s]", category: "ACE Medical")]
-	protected float m_fACE_Medical_SecondChanceFullRegenetationTimeS;
```

Comment:
This is still referenced in:
https://github.com/acemod/ACE-Anvil/blob/5828f1c28923b2880c86054aabc4c48b7689cf1c/addons/medical/scripts/Game/ACE_Medical/Components/Damage/SCR_CharacterDamageManagerComponent.c#L43

Reply:
Removed line 43 again. No clue how it how it came back in the first place.